client/Emote.lua

```diff
@@ -5,6 +5,7 @@ local ChosenDict = ""
 local ChosenAnimOptions = false
 local MovementType = 0
 local PlayerGender = "male"
+local PedHasProp = false
```

Comment:
Just want to know why this was added, doesn't seem related to translations

Reply:
Or was this missing from previous PR?

Reply:
This modification had to be made because the ped preview used the player's preview for the props